Prac07/guitarTest.py

```diff
@@ -0,0 +1,31 @@
+from Prac07.guitar import Guitar
+
+
+def main():
+
+    i = 0
+    guitars = []
+    while True:
+        name = input("Name: ")
+        if not name:
+            break
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitar_object = Guitar(name, year, cost)
+        guitars.append(guitar_object)
+        print(guitar_object)
+        i += 1
+
+    print("These are my guitars:")
+    for guitar_number in range(0,len(guitars)):
+        if guitars[guitar_number].is_vintage():
+            vintage_string = "(vintage)"
+        else:
+            vintage_string = ""
+
+        print("Guitar {}: {:>20} ({}), worth ${:10,.2f} {}".format(guitar_number + 1,
+                                                                           guitars[guitar_number].name,
```

Comment:
One final thing; there's a complaint from "PEP 8" for the main output, (this is irrelevant if you change as I've given examples of, but either way) I'm sure you know how to fix this, but encase you don't:
To fix; remove the entering down and space it out as if the "PEP 8" doesn't matter, get the light-bulb to appear and "Reformat file".